examples/global-aurora-mysql/main.tf

```diff
@@ -0,0 +1,155 @@
+provider "aws" {
+  region  = "eu-west-1"
+  version = ">= 3.14"
+}
+
+provider "aws" {
+  alias   = "primary"
+  region  = "eu-west-1"
+  version = ">= 3.14"
+}
+
+provider "aws" {
+  alias   = "secondary"
+  region  = "eu-west-2"
+  version = ">= 3.14"
+}
+
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+#####
+# VPC and subnets
+#####
+module "vpc_ireland" {
+  providers = {
+    aws = aws.primary
+  }
+
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 2.63"
+
+  name = "simple-vpc"
+
+  cidr = "10.0.0.0/16"
+
+  azs             = ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
+  private_subnets = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+  public_subnets  = ["10.0.101.0/24", "10.0.102.0/24", "10.0.103.0/24"]
+
+  enable_nat_gateway = false
+}
+
+module "vpc_london" {
+  providers = {
+    aws = aws.secondary
+  }
+
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 2.63"
+
+  name = "simple-vpc"
+
+  cidr = "10.0.0.0/16"
+
+  azs             = ["eu-west-2a", "eu-west-2b", "eu-west-2c"]
+  private_subnets = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+  public_subnets  = ["10.0.101.0/24", "10.0.102.0/24", "10.0.103.0/24"]
+
+  enable_nat_gateway = false
+}
+
+resource "aws_rds_global_cluster" "main" {
+  provider = aws.primary
+
+  engine                    = "aurora-mysql"
+  engine_version            = "5.7.mysql_aurora.2.08.0"
+  global_cluster_identifier = "main-global-mysql-cluster"
+  deletion_protection       = false
+
+  storage_encrypted = true
+
+  lifecycle {
+    ignore_changes = [engine_version]
+  }
+}
```

Comment:
Another assumption - We're adding support for RDS Global Clusters to be _added_ in later assuming you already have an RDS Cluster setup already. We're not creating a Global Cluster through the module itself?

Reply:
Example is being worked on :)